### PR TITLE
New package: gputils-1.5.0

### PR DESCRIPTION
**Pull request by lvitals**

### CODE REVIEW

File: srcpkgs/gputils/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'gputils'
+pkgname=gputils
+version=1.5.0
+revision=1
+build_style=gnu-configure
+makedepends="glibc-devel"
```
Comment (maxice8) [inline srcpkgs/gputils/template]: not necessary since glibc is part of base-devel

File: srcpkgs/gputils/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+depends="glibc"
```
Comment (maxice8) [inline srcpkgs/gputils/template]: not necessary

File: srcpkgs/gputils/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+short_desc="PIC Programming Utilities"
+maintainer="Leandro Vital <leavitals@gmail.com>"
+license="GPL-2.0"
```
Comment (maxice8) [inline srcpkgs/gputils/template]: please use a SPDX license identifier

File: srcpkgs/gputils/template [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+homepage="http://gputils.sourceforge.net/"
+distfiles="http://downloads.sourceforge.net/sourceforge/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum=f6a517c186b991f504be5e4585316871d5950568257885d37487bb368dc76227
+nostrip=yes
```
Comment (maxice8) [inline srcpkgs/gputils/template]: please justify

**Comment by maxice8** [comment]
On Sat, Jun 30, 2018 at 02:27:42PM -0700, Leandro Vital wrote:
> lvitals commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,12 @@
> +# Template file for 'gputils'
> +pkgname=gputils
> +version=1.5.0
> +revision=1
> +build_style=gnu-configure
> +short_desc="PIC Programming Utilities"
> +maintainer="Leandro Vital <leavitals@gmail.com>"
> +license="GPL-2.0-only"
> +homepage="http://gputils.sourceforge.net/"
> +distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
> +checksum=f6a517c186b991f504be5e4585316871d5950568257885d37487bb368dc76227
> +nostrip=yes
> 
> what is not necessary?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/542#discussion_r199332602

the notstrip=yes , what caused it's addition ?

File: srcpkgs/gputils/template [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+version=1.5.0
+revision=1
+build_style=gnu-configure
+short_desc="PIC Programming Utilities"
+maintainer="Leandro Vital <leavitals@gmail.com>"
+license="GPL-2.0-only"
+homepage="http://gputils.sourceforge.net/"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum=f6a517c186b991f504be5e4585316871d5950568257885d37487bb368dc76227
+nostrip=yes
```
[the remark this answers is not in the document]
Comment (lvitals) [reply srcpkgs/gputils/template]: no problem, removed

**Comment by maxice8** [comment]
squash all commits into one.

**Comment by lvitals** [comment]
@maxice8 I don't know how to do this

**Comment by maxice8** [comment]
$ git rebase -i upstream/master

replace all `pick` except the first one with `fixup`

**Pull request merged**